Change python virtualenv to /opt/mozdef/envs/python

This means we need to modify each cron/*.sh file to point to the new python binary.

@pwnbus I'm not able to build the image. when Irun `cd docker && sudo make build ` I'm getting following error
`make: *** No rule to make target `build'.  Stop.`

@pwnbus I'm getting some npm errors when running the build
```
node-pre-gyp ERR! Tried to download(404): https://github.com/kelektiv/node.bcrypt.js/releases/download/v1.0.3/bcrypt_lib-v1.0.3-node-v46-linux-x64.tar.gz 
node-pre-gyp ERR! Pre-built binaries not found for bcrypt@1.0.3 and node@4.7.0 (node-v46 ABI) (falling back to source compile with node-gyp) 
make: Entering directory `/node_modules/bcrypt/build'
  CXX(target) Release/obj.target/bcrypt_lib/src/blowfish.o
make: g++: Command not found
make: *** [Release/obj.target/bcrypt_lib/src/blowfish.o] Error 127
make: Leaving directory `/node_modules/bcrypt/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:276:23)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:211:12)
gyp ERR! System Linux 4.4.0-92-generic
gyp ERR! command "/usr/bin/node" "/usr/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "build" "--fallback-to-build" "--module=/node_modules/bcrypt/lib/binding/bcrypt_lib.node" "--module_name=bcrypt_lib" "--module_path=/node_modules/bcrypt/lib/binding"
gyp ERR! cwd /node_modules/bcrypt
gyp ERR! node -v v4.7.0
gyp ERR! node-gyp -v v3.4.0
gyp ERR! not ok 
node-pre-gyp ERR! build error 
node-pre-gyp ERR! stack Error: Failed to execute '/usr/bin/node /usr/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js build --fallback-to-build --module=/node_modules/bcrypt/lib/binding/bcrypt_lib.node --module_name=bcrypt_lib --module_path=/node_modules/bcrypt/lib/binding' (1)
node-pre-gyp ERR! stack     at ChildProcess.<anonymous> (/node_modules/bcrypt/node_modules/node-pre-gyp/lib/util/compile.js:83:29)
node-pre-gyp ERR! stack     at emitTwo (events.js:87:13)
node-pre-gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
node-pre-gyp ERR! stack     at maybeClose (internal/child_process.js:854:16)
node-pre-gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:222:5)
node-pre-gyp ERR! System Linux 4.4.0-92-generic
node-pre-gyp ERR! command "/usr/bin/node" "/node_modules/bcrypt/node_modules/.bin/node-pre-gyp" "install" "--fallback-to-build"
node-pre-gyp ERR! cwd /node_modules/bcrypt
node-pre-gyp ERR! node -v v4.7.0
node-pre-gyp ERR! node-pre-gyp -v v0.6.36
node-pre-gyp ERR! not ok 
Failed to execute '/usr/bin/node /usr/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js build --fallback-to-build --module=/node_modules/bcrypt/lib/binding/bcrypt_lib.node --module_name=bcrypt_lib --module_path=/node_modules/bcrypt/lib/binding' (1)
npm ERR! Linux 4.4.0-92-generic
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "install" "source-map-support@0.4.2" "semver@5.3.0" "fibers@1.0.13" "amdefine@1.0.0" "underscore@1.8.3" "bcrypt"
npm ERR! node v4.7.0
npm ERR! npm  v2.15.11
npm ERR! code ELIFECYCLE

npm ERR! bcrypt@1.0.3 install: `node-pre-gyp install --fallback-to-build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the bcrypt@1.0.3 install script 'node-pre-gyp install --fallback-to-build'.
npm ERR! This is most likely a problem with the bcrypt package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-pre-gyp install --fallback-to-build
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs bcrypt
npm ERR! Or if that isn't available, you can get their info via:
npm ERR! 
npm ERR!     npm owner ls bcrypt
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /npm-debug.log

```

--- a/cron/auth02mozdef.sh
+++ b/cron/auth02mozdef.sh
@@ -6,5 +6,5 @@
 # Author: gdestuynder@mozilla.com
 
 
-source  /opt/mozdef/envs/mozdef/bin/activate
+source  /opt/mozdef/envs/python/bin/activate
 /opt/mozdef/envs/mozdef/cron/auth02mozdef.py

--- a/cron/backupSnapshot.sh
+++ b/cron/backupSnapshot.sh
@@ -5,5 +5,5 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 # Copyright (c) 2014 Mozilla Corporation
 
-source  /opt/mozdef/envs/mozdef/bin/activate
+source  /opt/mozdef/envs/python/bin/activate
 /opt/mozdef/envs/mozdef/cron/backupSnapshot.py -c /opt/mozdef/envs/mozdef/cron/backup.conf

--- a/cron/collectAttackers.sh
+++ b/cron/collectAttackers.sh
@@ -5,5 +5,5 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 # Copyright (c) 2014 Mozilla Corporation
 
-source  /opt/mozdef/envs/mozdef/bin/activate
+source  /opt/mozdef/envs/python/bin/activate
 /opt/mozdef/envs/mozdef/cron/collectAttackers.py -c /opt/mozdef/envs/mozdef/cron/collectAttackers.conf

--- a/cron/collectSSHFingerprints.sh
+++ b/cron/collectSSHFingerprints.sh
@@ -5,6 +5,6 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 # Copyright (c) 2014 Mozilla Corporation
 
-source  /opt/mozdef/envs/mozdef/bin/activate
+source  /opt/mozdef/envs/python/bin/activate
 /opt/mozdef/envs/mozdef/cron/collectSSHFingerprints.py -c /opt/mozdef/envs/mozdef/cron/collectSSHFingerprints.conf
 

--- a/cron/correlateUserMacAddress.sh
+++ b/cron/correlateUserMacAddress.sh
@@ -5,6 +5,6 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 # Copyright (c) 2014 Mozilla Corporation
 
-source  /opt/mozdef/envs/mozdef/bin/activate
+source  /opt/mozdef/envs/python/bin/activate
 /opt/mozdef/envs/mozdef/cron/correlateUserMacAddress.py -c /opt/mozdef/envs/mozdef/cron/correlateUserMacAddress.conf
 

--- a/cron/createIPBlockList.sh
+++ b/cron/createIPBlockList.sh
@@ -5,6 +5,6 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 # Copyright (c) 2014 Mozilla Corporation
 
-source  /opt/mozdef/envs/mozdef/bin/activate
+source  /opt/mozdef/envs/python/bin/activate
 /opt/mozdef/envs/mozdef/cron/createIPBlockList.py -c /opt/mozdef/envs/mozdef/cron/createIPBlockList.conf
 

--- a/cron/duo_logpull.sh
+++ b/cron/duo_logpull.sh
@@ -5,5 +5,5 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 # Copyright (c) 2014 Mozilla Corporation
 
-source  /opt/mozdef/envs/mozdef/bin/activate
+source  /opt/mozdef/envs/python/bin/activate
 /opt/mozdef/envs/mozdef/cron/duo_logpull.py -c /opt/mozdef/envs/mozdef/cron/duo_logpull.conf

--- a/cron/duo_logpull_releng.sh
+++ b/cron/duo_logpull_releng.sh
@@ -5,6 +5,6 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 # Copyright (c) 2014 Mozilla Corporation
 
-source /opt/mozdef/envs/mozdef/bin/activate
+source /opt/mozdef/envs/python/bin/activate
 /opt/mozdef/envs/mozdef/cron/duo_logpull.py -c /opt/mozdef/envs/mozdef/cron/duo_logpull_releng.conf
 

--- a/cron/esMaint.sh
+++ b/cron/esMaint.sh
@@ -5,6 +5,6 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 # Copyright (c) 2014 Mozilla Corporation
 
-source  /opt/mozdef/envs/mozdef/bin/activate
+source  /opt/mozdef/envs/python/bin/activate
 /opt/mozdef/envs/mozdef/cron/rotateIndexes.py -c /opt/mozdef/envs/mozdef/cron/backup.conf
 

--- a/cron/eventStats.sh
+++ b/cron/eventStats.sh
@@ -5,5 +5,5 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 # Copyright (c) 2014 Mozilla Corporation
 
-source  /opt/mozdef/envs/mozdef/bin/activate
+source  /opt/mozdef/envs/python/bin/activate
 /opt/mozdef/envs/mozdef/cron/eventStats.py -c /opt/mozdef/envs/mozdef/cron/eventStats.conf

--- a/cron/google2mozdef.sh
+++ b/cron/google2mozdef.sh
@@ -5,6 +5,6 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 # Copyright (c) 2014 Mozilla Corporation
 
-source  /opt/mozdef/envs/mozdef/bin/activate
+source  /opt/mozdef/envs/python/bin/activate
 /opt/mozdef/envs/mozdef/cron/google2mozdef.py -c /opt/mozdef/envs/mozdef/cron/google2mozdef.conf
 

--- a/cron/healthAndStatus-fxa.sh
+++ b/cron/healthAndStatus-fxa.sh
@@ -5,5 +5,5 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 # Copyright (c) 2014 Mozilla Corporation
 
-source  /opt/mozdef/envs/mozdef/bin/activate
+source  /opt/mozdef/envs/python/bin/activate
 /opt/mozdef/envs/mozdef/cron/healthAndStatus.py -c /opt/mozdef/envs/mozdef/cron/healthAndStatus.fxa.conf

--- a/cron/healthAndStatus-mdc1.sh
+++ b/cron/healthAndStatus-mdc1.sh
@@ -5,5 +5,5 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 # Copyright (c) 2014 Mozilla Corporation
 
-source  /opt/mozdef/envs/mozdef/bin/activate
+source  /opt/mozdef/envs/python/bin/activate
 /opt/mozdef/envs/mozdef/cron/healthAndStatus.py -c /opt/mozdef/envs/mozdef/cron/healthAndStatus.mdc1.conf

--- a/cron/healthAndStatus.sh
+++ b/cron/healthAndStatus.sh
@@ -5,5 +5,5 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 # Copyright (c) 2014 Mozilla Corporation
 
-source  /opt/mozdef/envs/mozdef/bin/activate
+source  /opt/mozdef/envs/python/bin/activate
 /opt/mozdef/envs/mozdef/cron/healthAndStatus.py -c /opt/mozdef/envs/mozdef/cron/healthAndStatus.conf

--- a/cron/healthToMongo.sh
+++ b/cron/healthToMongo.sh
@@ -5,5 +5,5 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 # Copyright (c) 2014 Mozilla Corporation
 
-source  /opt/mozdef/envs/mozdef/bin/activate
+source  /opt/mozdef/envs/python/bin/activate
 /opt/mozdef/envs/mozdef/cron/healthToMongo.py -c /opt/mozdef/envs/mozdef/cron/healthToMongo.conf

--- a/cron/import_threat_exchange.sh
+++ b/cron/import_threat_exchange.sh
@@ -5,5 +5,5 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 # Copyright (c) 2017 Mozilla Corporation
 
-source  /opt/mozdef/envs/mozdef/bin/activate
+source  /opt/mozdef/envs/python/bin/activate
 /opt/mozdef/envs/mozdef/cron/import_threat_exchange.py -c /opt/mozdef/envs/mozdef/cron/import_threat_exchange.conf

--- a/cron/okta2mozdef.sh
+++ b/cron/okta2mozdef.sh
@@ -5,6 +5,6 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 # Copyright (c) 2014 Mozilla Corporation
 
-source  /opt/mozdef/envs/mozdef/bin/activate
+source  /opt/mozdef/envs/python/bin/activate
 /opt/mozdef/envs/mozdef/cron/okta2mozdef.py -c /opt/mozdef/envs/mozdef/cron/okta2mozdef.conf
 

--- a/cron/pruneES.sh
+++ b/cron/pruneES.sh
@@ -5,6 +5,6 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 # Copyright (c) 2014 Mozilla Corporation
 
-source  /opt/mozdef/envs/mozdef/bin/activate
+source  /opt/mozdef/envs/python/bin/activate
 /opt/mozdef/envs/mozdef/cron/pruneIndexes.py -c /opt/mozdef/envs/mozdef/cron/backup.conf
 

--- a/cron/syncAlertsToMongo.sh
+++ b/cron/syncAlertsToMongo.sh
@@ -5,5 +5,5 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 # Copyright (c) 2014 Mozilla Corporation
 
-source  /opt/mozdef/envs/mozdef/bin/activate
+source  /opt/mozdef/envs/python/bin/activate
 /opt/mozdef/envs/mozdef/cron/syncAlertsToMongo.py -c /opt/mozdef/envs/mozdef/cron/syncAlertsToMongo.conf

--- a/cron/update_generic_alerts.sh
+++ b/cron/update_generic_alerts.sh
@@ -5,5 +5,5 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 # Copyright (c) 2017 Mozilla Corporation
 
-source  /opt/mozdef/envs/mozdef/bin/activate
+source  /opt/mozdef/envs/python/bin/activate
 /opt/mozdef/envs/mozdef/cron/update_generic_alerts.py -c /opt/mozdef/envs/mozdef/cron/update_generic_alerts.conf

--- a/cron/update_geolite_db.sh
+++ b/cron/update_geolite_db.sh
@@ -5,5 +5,5 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 # Copyright (c) 2017 Mozilla Corporation
 
-source  /opt/mozdef/envs/mozdef/bin/activate
+source  /opt/mozdef/envs/python/bin/activate
 /opt/mozdef/envs/mozdef/cron/update_geolite_db.py -c /opt/mozdef/envs/mozdef/cron/update_geolite_db.conf

--- a/cron/update_ip_list.sh
+++ b/cron/update_ip_list.sh
@@ -5,5 +5,5 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 # Copyright (c) 2017 Mozilla Corporation
 
-source  /opt/mozdef/envs/mozdef/bin/activate
+source  /opt/mozdef/envs/python/bin/activate
 /opt/mozdef/envs/mozdef/cron/update_ip_list.py -c /opt/mozdef/envs/mozdef/cron/update_ip_list.conf

--- a/cron/verify_event_fields.sh
+++ b/cron/verify_event_fields.sh
@@ -5,5 +5,5 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 # Copyright (c) 2017 Mozilla Corporation
 
-source  /opt/mozdef/envs/mozdef/bin/activate
+source  /opt/mozdef/envs/python/bin/activate
 /opt/mozdef/envs/mozdef/cron/verify_event_fields.py -c /opt/mozdef/envs/mozdef/cron/verify_event_fields.conf

--- a/cron/vidyo2MozDef.sh
+++ b/cron/vidyo2MozDef.sh
@@ -5,6 +5,6 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 # Copyright (c) 2014 Mozilla Corporation
 
-source  /opt/mozdef/envs/mozdef/bin/activate
+source  /opt/mozdef/envs/python/bin/activate
 /opt/mozdef/envs/mozdef/cron/vidyo2MozDef.py -c /opt/mozdef/envs/mozdef/cron/vidyo2MozDef.conf
 

